census_view.cpp

```diff
@@ -1317,6 +2184,62 @@ void CensusView::update_visible_columns()
         }
 }
 
+void CensusGridView::update_visible_columns()
+{
+    // Display exactly those columns whose rows aren't all identical. For
+    // this purpose, consider as "rows" the individual cells--and also the
+    // case and class defaults, even though they aren't displayed in rows.
+    // Reason: although the case and class defaults are hidden, they're
+    // still information--so if the user made them different from any cell
+    // wrt some column, we respect that conscious decision.
+    auto const& all_headers = case_parms()[0].member_names();
```

Comment:
```suggestion
    std::vector<std::string> const& all_headers = case_parms()[0].member_names();
```
Just to avoid diff in #121.